operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationServiceProvider.java

```diff
@@ -2,6 +2,9 @@
 
 import java.util.function.Consumer;
 
+/**
+ * For internal usage only. To access operator configuration through the code.
```

Comment:
```suggestion
 * For internal usage only, to avoid passing the operator configuration around. Preferred way to get to the ConfigurationService is via the reconciliation context.
```